src/OrbitGl/App.cpp

```diff
@@ -993,8 +993,9 @@ ErrorMessageOr<void> OrbitApp::OnSaveCapture(const std::filesystem::path& file_n
 }
 
 static ErrorMessageOr<CaptureListener::CaptureOutcome> LoadCaptureFromNewFormat(
-    CaptureListener* listener, CaptureFile* capture_file,
+    const std::filesystem::path& file_path, CaptureListener* listener, CaptureFile* capture_file,
```

Comment:
Can `CaptureFile` have a method that exposes `CaptureFileImpl::file_path_` or is there a reason against it?

It seems redundant to pass both `file_path` (only for logging reasons) and the respective `CaptureFile`.